terraform/monitoring/main.tf

```diff
@@ -592,12 +592,11 @@ resource "grafana_dashboard" "at_a_glance" {
             }
           ],
           "executionErrorState" : "alerting",
-          "for" : "5m",
-          "frequency" : "1m",
+          "frequency" : "5m",
```

Comment:
Why reduce frequency to every 5m? Maybe we can leave at 1m?

Reply:
I think 5m for alerting frequency is enough to catch issues if they happen, but we can use 1m if we need a lower threshold. I've changed it to 1m.